### PR TITLE
adds dqedit

### PR DESCRIPTION
**Pull request by JenSeReal**
Adds dqedit to dqualizer main repo

- - -
This change is [Reviewable](https://reviewable.io/reviews/dqualizer/dqualizer/38)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: "3.9"
 services:
   ddd-autohaus:
-    image: ghcr.io/dqualizer/dqsamples-ddd-autohaus
+    image: ghcr.io/dqualizer/ddd-autohaus:main
```
Comment (EddeCCC) [inline docker-compose.yml:4]: Das neue Image lässt sich nicht ziehen (Unauthorized)

**Review by EddeCCC — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -139,3 +150,4 @@ services:
       - DOCKER_INFLUXDB_INIT_ADMIN_TOKEN
     depends_on:
       - influxdb
+    attach: false
```
Comment (levinkerschberger) [inline docker-compose.yml:153]: I´m getting an error here: 
_services.dqDatabase Additional property attach is not allowed_
The error message seems to rotate between the services, when I retry to compose up.

File: docker-compose.yml (hunk 1 above)
Comment (levinkerschberger) [reply docker-compose.yml:4]: Same for me, when I try to pull separatly.

Comment (JenSeReal) [reply docker-compose.yml:4]: Die Visibility war noch auf private, anscheinend bin ich Lokal noch angemeldet und deswegen hat es funktioniert :D. Hab die Visibility auf Public gestellt 👍

File: .env [hunk 3]
```diff
@@ -3,15 +3,14 @@
...
 DQTRANSLATOR_PORT="8080"
 DQEXEC_PORT="8090"
-MONGO_PORT="27017"
 MONGO_MANAGEMENT_PORT="8081"
 INFLUX_PORT="8086"
 GRAFANA_PORT="3030"
 SAMPLE_APPLICATION_PORT="9000"
-RABBIT_PORT="15672"
-RABBIT_MANAGEMENT_PORT="5672"
+RABBIT_MANAGEMENT_PORT="15672"
```
Comment (levinkerschberger) [inline .env:13]: Seems like there was a mistake with the rabbit-port.
Why did you delete the Mongo- and Rabbit-Ports?

**Review by EddeCCC — APPROVED** [review]

File: .env (hunk 3 above)
Comment (JenSeReal) [reply .env:13]: Rabbit and Mongo don't need to be exposed, since they don't need to be accessed from outside of docker (just like dqexec, dqtranslator...). I only want to expose the management ui - for that thats the right port

File: docker-compose.yml (hunk 2 above)
Comment (JenSeReal) [reply docker-compose.yml:153]: https://docs.docker.com/engine/reference/commandline/compose_up/ It is a real property.. it allows to not show logs of specific container when running `docker compose up`

File: .env (hunk 3 above)
Comment (levinkerschberger) [reply .env:13]: Makes sense!:)

**Review by levinkerschberger — APPROVED** [review]

File: docker-compose.yml [hunk 4]
```diff
@@ -37,6 +37,11 @@ services:
       retries: 5
       timeout: 10s
 
+  dqEdit:
+    image: ghcr.io/dqualizer/dqedit-react:react-dev
```
Comment (franksn90) [inline docker-compose.yml:41]: As agreed today, let's aim at a stable release version before officially releasing the new showacse version.

**Review by franksn90 — CHANGES_REQUESTED** [review]
Release postponed until Hackathon
